### DISCUSSION TITLE
(Tutorial) Add button for SSO that works even with Android

### DISCUSSION THREAD

**Discussion by M4TH1EU** [9p4/jellyfin-plugin-sso#228] [General]
Add in General -> Branding ->  Login disclaimer :
```html
<form action="https://jellyfin.<your domain>.com/sso/OID/start/<your provider name>">
  <button class="raised block emby-button button-submit" 
          onclick="if (!localStorage.getItem('_deviceId2') && window.NativeShell?.AppHost?.deviceId) { localStorage.setItem('_deviceId2', window.NativeShell.AppHost.deviceId()); }" 
          target="_self" 
          style="display: flex; align-items: center; justify-content: center; gap: 10px; padding: 12px 20px; font-size: 16px;">
    <span class="material-icons shield" aria-hidden="true"></span>
    <span>Connect using SSO</span>
  </button>
</form>
```

![image](https://github.com/user-attachments/assets/c10e15b1-fc9b-445e-b7f6-8655c946ddcc)

Thanks to : https://github.com/9p4/jellyfin-plugin-sso/issues/61#issuecomment-1820045126

**Comment by kesavananbu** [comment]
Do you know if this is working for you? 

loginDisclaimer.innerHTML = DOMPurify.sanitize(markdownIt({ html: true }).render(options.LoginDisclaimer || '')); 
This line will clean all the stuff like clicks or scripts in branding.
